pvextractor/geometry/helpers.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord, UnitSphericalRepresentation, CartesianRepresentation
+
+from .path import Path
+
+
+class PathFromCenter(Path):
+    """
+    A simple path defined by a center, length, and position angle.
+
+    Parameters
+    ----------
+    center : `~astropy.coordinates.SkyCoord`
+        The center of the path
+    length : `~astropy.units.Quantity`
+        The length of the path in angular units
+    angle : `~astropy.units.Quantity`
+        The position angle of the path, counter-clockwise
+    sample : int
+        How many points to sample along the path. By default, this is 2 (the
+        two end points. For small fields of view, this will be a good
+        approximation to the path, but for larger fields of view, where
+        spherical distortions become important, this should be increased to
+        provide a smooth path.
+    width : None or float or :class:`~astropy.units.Quantity`
+        The width of the path. If ``coords`` is passed as a list of pixel
+        positions, the width should be given (if passed) as a floating-point
+        value in pixels. If ``coords`` is a coordinate object, the width
+        should be passed as a :class:`~astropy.units.Quantity` instance with
+        units of angle.
+
+    Notes
+    -----
+    The orientation of the final path will be such that for a position angle of
+    zero, the path goes from North to South. For a position angle of 90
```

Comment:
This seems like a strange convention.  I'm used to zero either pointing east->west or south->north.  Isn't position angle defined with 0 = east->west?